package.json

```diff
@@ -29,7 +29,7 @@
   "dependencies": {
     "@embroider/macros": "^1.6.0",
     "@embroider/util": "^1.6.0",
-    "@lblod/submission-form-helpers": "^1.3.0",
+    "@lblod/submission-form-helpers": "https://github.com/lblod/submission-form-helpers#262d0c1511f4d3502eb75f73f02b43a6aa11b3d0",
```

Comment:
I guess we can merge the submission-form-helpers PR, release a new version and use that here before merging this one?